chart/skywalking/values.yaml

```diff
@@ -19,6 +19,9 @@
 
 serviceAccounts:
   oap:
+    # By default, create SkyWalking's ServiceAccount. If set to false, you also need to change `serviceAccounts.oap.name` value to a custom ServiceAccount name.
+    create: true
+    name: ""
```

Comment:
Can we just keep 

```yaml
  serviceAccounts:
    oap: "" # If this is set to empty, SkyWalking creates the ServiceAccount, if you change `serviceAccounts.oap` value to a custom ServiceAccount name, SkyWalking won't create ServiceAccount, but please make sure the given ServiceAccount has correct permissions.

Reply:
```
serviceAccounts:
  oap:
    # By default, create SkyWalking's ServiceAccount. If set to false, you also need to change `serviceAccounts.oap.name` value to a custom ServiceAccount name.
    create: true
    name: ""
```
I think this is more convenient. When we deploy multiple skywalking clusters in one k8s at the same time, multiple clusterRole may be created. At this time, we can use the name field to distinguish different SW clusterRole, so as not to cause clusterRole conflicts.